include/irr/builtin/material_compiler/glsl/common.glsl

```diff
@@ -673,241 +500,442 @@ void runNormalPrecompStream(in instr_stream_t stream, in mat2 dUV)
 	}
 }
 
-void flipCurrInteraction()
+//TODO move or rename those
+//ggx
+float irr_glsl_ggx_height_correlated_cos_eval_DG(in float NdotH, in float NdotL, in float NdotV, in float a2)
```

Comment:
if you heed my advice... you'll no longer need all these
https://github.com/buildaworldnet/IrrlichtBAW/pull/554/files#r526504617